torch/utils/data/__init__.py

```diff
@@ -39,6 +37,13 @@
     runtime_validation,
     runtime_validation_disabled,
 )
+from torch.utils.data._utils.collate import (
+    default_collate,
+    default_convert,
+    default_collate_fn_map,
+    register_default_collate_for,
```

Comment:
We currently provide the first two of these through `torch.utils.data.dataloader`. I think the CI will break if they are not routed through there

Reply:
Yes... I noticed this at the `Registry` version. 
Should we include them here or pass through `torch.utils.data.dataloader`?